src/infra/email/views/sendStudentQRCodeEmail.js

```diff
@@ -12,17 +18,24 @@ async function sendStudentQRCodeEmail(studentData) {
     from: process.env.EMAIL_USER,
     to: studentData.email,
     subject: 'QRCode',
```

Comment:
Modificar título do e-mail para algo mais user friendly.

```suggestion
    subject: 'Aqui está seu QR Code para receber a merenda!',
```